fabmukunzi/my-brand
# #iG30lIE0 implementing landing-about-skills-portfolio-contact-and-login-page

#### What does this PR do?
This PR implements landing,about,skills,portfolio,contact and login page
#### Description of Task to be completed?
-Add Homepage
-Add about page
-Add Skills page
_Add Portfolio page
-Add contact page
-Add login page

#### How should this be manually tested?
_Clone the [repo](https://github.com/fabmukunzi/my-brand)
-Checkout the branch feat-implement-landing-about-skills-portfolio-contact-and-login-page-iG30lIE0
-Open index.html in your browser to check the pages

#### What are the relevant pivotal tracker stories?
[#iG30lIE0](https://trello.com/b/iG30lIE0/kanban-template)
#### Screenshots (if appropriate)
![skils](https://user-images.githubusercontent.com/83646585/202885270-a5f77945-0116-46fe-b303-3e3751787f8b.PNG)

#### Questions:
